### PR TITLE
NEWS update for 2.0.0 rc1

### PR DESCRIPTION
**Pull request by hppritcha**
Fixes open-mpi/ompi#1172

@jsquyres first cut.  

Signed-off-by: Howard Pritchard howardp@lanl.gov

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1162/ for details.

File: NEWS [hunk 1]
```diff
@@ -66,6 +66,115 @@ Master (not on release branches yet)
...
+- Remove --enable-dist configure option
+- Remove support for Cray XT systems
+- Numerous fixes/improvements to orte-dvm.  Special
+  thanks to Mark Santcroos for his help.
+- Silence a compiler warning in PSM MTL.  Thanks
+  to Adrian Reber for reporting this.
+- Add missing Fortran bindings for MPI_Win_allocate.
+  Thanks to Christoph Niethammer for reporting
+  and fixing.
+- replace use of alloca with malloc for certain
```
Comment (jsquyres) [inline NEWS]: _R_eplace

**Comment by jsquyres** [comment]
Some minor comments:
- There isn't a whole lot of emphasis here on new things that have been added in 2.0.0 -- most of the bullets are about fixes.  We should probably have some high-level comments about big features that were added.
- It looks like many of the bullets are formatted for (much) less than ~75 columns or so.  Do you want to re-flow for line breaks around 75/76 chars?
- There's a few bullets where it might be useful to group them together.  E.g.:

```
* Changes to configure CLI options:
  * Add --with-platform-patches-dir
  * Add a --with-pmi-libdir configure option for environments that install...
```

Honestly, probably only my first comment is the important one.  The 2nd / 3rd are my personal IMHO...

File: NEWS [hunk 2]
```diff
@@ -66,6 +66,115 @@ Master (not on release branches yet)
...
+- Performance improvements to MPI RMA
+- Removed checkpoint/restart code due to loss of maintainer :-(
+- Add a new MCA parameter - opal_common_verbs_want_fork_support. This replaces
+  the btl_openib_want_fork_support parameter.
+- Add a -tune command line option to simplify setting many environment
+  variables and mca parameters.
+- Removed --enable-mpi-profiling configure option.
+- Add --with-platform-patches-dir
+- Add a new mca parameter default-dash-to-host to offer a mirror of the default_hostfile.
+- Add a  --with-pmi-libdir configure option for environments that install
```
Comment (jsquyres) [inline NEWS]: Extra space after "Add a"

File: NEWS [hunk 3]
```diff
@@ -66,6 +66,115 @@ Master (not on release branches yet)
...
+  PMI libs in a non-default location
+- Add the ability to specify the number of desired slots in the --host option.
+- Remove support for OS X Leopard
+- Remove --enable-dist configure option
```
Comment (jsquyres) [inline NEWS]: `--enable-dist` was always an internal optional, anyway -- probably don't need to list it here.

**Comment by hppritcha** [comment]
@jsquyres second cut.  more verbiage about new stuff - based on SC 15 BOF slides, etc. plus
cleanups per your feedback.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1170/ for details.

File: NEWS [hunk 4]
```diff
@@ -66,6 +66,111 @@ Master (not on release branches yet)
...
+- Numerous fixes/improvements to orte-dvm.  Special thanks to Mark Santcroos
+  for his help.
+- Silence a compiler warning in PSM MTL.  Thanks to Adrian Reber for reporting this.
+- Add missing Fortran bindings for MPI_Win_allocate.  Thanks to Christoph
+  Niethammer for reporting and fixing.
+- Replace use of alloca with malloc for certain datatype creation functions.
+  Thanks to Bogdan Sataric for reporting this.
+- Fix use of MPI_LB and MPI_UB in creation of of certain MPI datatypes.
+  Thanks to Gus Correa
+  for helping to fix this.
```
Comment (jsquyres) [inline NEWS]: probably should merge these 2 short lines

File: NEWS [hunk 5]
```diff
@@ -66,6 +66,111 @@ Master (not on release branches yet)
...
+- Implement a workaround for a libtool problem.  Thanks to Eric Schnetter for
+  reporting and fixing.
+- Improve hcoll library detection in configure.  Thanks to David Shrader and
+  Ake Sandgren for
```
Comment (jsquyres) [inline NEWS]: probably should merge these 2 short lines

File: NEWS [hunk 6]
```diff
@@ -66,6 +66,111 @@ Master (not on release branches yet)
...
+  and Siegmar Gross for reporting this issue.
+- Fix a performance problem for large messages for Cray XC systems.  Thanks to Jerome
+  Vienne for reporting this.
+- Fix an issue with MPI_Win_lock_all.  Thanks to Thomas Jahns for reporting.
+- Fix an issue with passing a parameter to configure multiple times.  Thanks to
+  QuesarVII for reporting and supplying a fix.
+- Add support for ALPS resource allocation system on Cray CLE 5.2 and later.
+  Thanks to Mark Santcroos.
+- Correction to HACKING file.  Thanks to Maximilien
+  Levesque.
```
Comment (jsquyres) [inline NEWS]: Merge "Levesque" up a line?

**Comment by jsquyres** [comment]
Minor nit picks, but otherwise looks good.

**Comment by hppritcha** [comment]
i'll fix the nits.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ompi-release-pr/1171/ for details.

**Comment by jsquyres** [comment]
:+1:

**Pull request merged**